gbpservice/nfp/service_vendor_agents/vyos/oc_config_server/server2.py

```diff
@@ -460,6 +461,18 @@ def del_rule():
         return json.dumps(dict(status=True))
 
 
+@app.route('/configure-rsyslog-as-client', methods=['POST'])
```

Comment:
In non-commercial version, configuration of syslog will not land on the VM agent. We should keep this API separate and load only when visibility components are installed.